src/inmanta/env.py

```diff
@@ -104,7 +104,11 @@ def get_installed_packages(self, only_editable: bool = False) -> Dict[str, versi
         return {r["name"]: version.Version(r["version"]) for r in json.loads(output)}
 
     def install_from_index(
-        self, requirements: List[Requirement], index_urls: Optional[List[str]] = None, upgrade: bool = False
+        self,
+        requirements: List[Requirement],
+        index_urls: Optional[List[str]] = None,
+        upgrade: bool = False,
+        allow_pre_releases: bool = False,
```

Comment:
Does inmanta projects install use this flag correctly?  (or is that put in the relevant ticket?)

Reply:
I don't think it does and there doesn't seem to be a ticket for that. I was planning to create a ticket, but maybe it's easier to include it in this PR. I will make the change.

I will also add some user documentation on how the `install_mode` option is interpreted for V2 modules. That's missing as well.